console/src/assets/i18n/de.json

```diff
@@ -640,7 +640,7 @@
     },
     "ADDTRIGGER": "Trigger hinzufügen",
     "TIMEOUT": "Timeout",
-    "TIMEOUTINSEC": "Timout in Sekunden",
+    "TIMEOUTINSEC": "Timeout in Sekunden",
```

Comment:
Timeout is not a proper German word, or am I wrong?

Reply:
As technical term fine, imo